### DISCUSSION TITLE
some questions about the "dpgen init_reaction"

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/dpgen#1045] [Q&A]
Rencently I want to use the  "init_reaction" to generate some initial conformations, when setting the machine.json, there is a little reference documents for me，now I guess it that the "reaxff", "build" and "fp"  are essential in the machine.json, but when I write the information in "build", I don‘t know how to set the "command", by reading the code, I guess that it may create the input file for ab initio calculation, but what is the command, if I need to install other software? can you give me a finished machine.json? thank you very much.

**Comment by HuangJiameng** [comment]
You can refer to https://docs.deepmodeling.com/projects/dpgen/en/devel/init/init-reaction.html

**Comment by Anopaul** [comment]
The 'init_reaction' command is used to generate the initial dataset for the afterward training processes, partly the same as 'init_bulk' or 'init_surf'. The difference of it is that this command is suitable for small organic molecules like alkanes. When you would like to use the 'init_reaction' command, you need to install another software named  'mddatasetbuilder' (https://github.com/tongzhugroup/mddatasetbuilder)' for the 'build' part.  Below is a sample for you to refer to.
[machine.zip](https://github.com/deepmodeling/dpgen/files/10084692/machine.zip)

**Comment by cui-hq** [comment]
thank you very much发自我的荣耀手机-------- 原始邮件 --------发件人： Anopaul ***@***.***>日期： 2022年11月24日周四 20:52收件人： deepmodeling/dpgen ***@***.***>抄送： cui-hq ***@***.***>, Author ***@***.***>主    题： Re: [deepmodeling/dpgen] some questions about the "dpgen init_reaction" (Discussion #1045)
The 'init_reaction' command is used to generate the initial dataset for the afterward training processes, partly the same as 'init_bulk' or 'init_surf'. The difference of it is that this command is suitable for small organic molecules like alkanes. When you would like to use the 'init_reaction' command, you need to install another software named  'mddatasetbuilder' (https://github.com/tongzhugroup/mddatasetbuilder)' for the 'build' part.  Below is a sample for you to refer to.
machine.zip
